### ISSUE TITLE
istanbuljs / nyc test coverage report validation along with individual files with different configuration

### ISSUE THREAD

**Issue by artbindu** [istanbuljs/nyc#1491]
This is not an issue, needs some confirmation about nyc configuration.

 Is there any way to check all individual file coverage reports along with the consolated coverage report through `.nycrc.json` ?
 
 As an example: 
  My consolated coverage report should pass 80% and the individual files coverage report should pass 65%.

For the consolated coverage report I have to add the following configuration at `.nycrc.json` file.
```json
    {
      "statements": 80,
      "branches": 80,
      "functions": 80,
      "lines": 80
    }
```
   How do I check the individual files coverage report with 65% coverage?
    
    
Details:
https://stackoverflow.com/questions/73769935/istanbuljs-nyc-test-coverage-report-validation-along-with-individual-files-wit

**Comment by mahanthi-arun-kumar** [comment]
@artbindu  did you get any solution for this?

**Comment by artbindu** [comment]
@mahanthi-arun-kumar 
No, I did not see anyupdate reguarding this in the [istanbul/nyc](https://github.com/istanbuljs/nyc) documentation.

**Comment by varunrajasekhar** [comment]
Me too!! Looking for a fix... sadly found nothing :(
